src/javascript/app/pages/trade/charts/digit_info.js

```diff
@@ -246,6 +246,10 @@ const DigitInfo = (() => {
     };
 
     const updateChart = async (tick) => {
+        if (typeof chart === 'undefined') {
```

Comment:
![image](https://github.com/deriv-com/smarttrader/assets/56330681/d6cf3059-2683-4245-a47e-f0338ed67f01)
Same logic exists in the update function